### PR TITLE
Make flattenMode available from the CLI Feature request #67

### PR DESCRIPTION
**Pull request by andye2004**

### CODE REVIEW

**Comment by matthiasbalke** [comment]
@andye2004 how about `flatten.mode` as an commandline argument name? I was thinking about adding a `flatten.skip` configuration, to be able to skip the plugin execution like other plugins support it. This way the command line arguments would be grouped by the plugin name `flatten`.

What do you think?

**Comment by andye2004** [comment]
@matthiasbalke Hi Matthias, happy to swap over. Code / PR modified as requested.

File: src/test/java/org/codehaus/mojo/flatten/CmdLinePropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.codehaus.mojo.flatten;
+
+import org.apache.maven.plugin.testing.AbstractMojoTestCase;
+import org.junit.Test;
+
+public class CmdLinePropertiesTest extends AbstractMojoTestCase {
+    @Test
+    public void test() throws Exception {
+//        System.out.println("Trying to look up mojo");
```
Comment (matthiasbalke) [inline src/test/java/org/codehaus/mojo/flatten/CmdLinePropertiesTest.java]: Please avoid commented code.  Pleae fix this test before merging.

**Comment by andye2004** [comment]
Sorry @matthiasbalke, stealth commit. Sneaked in accidentally. Guess that's why we have PRs and reviews :)

**Comment by matthiasbalke** [comment]
@andye2004 yeah reviews FTW

**Comment by andye2004** [comment]
@matthiasbalke Hi Matthias, just wondering if you knew when this PR would be pulled in?

**Comment by matthiasbalke** [comment]
@andye2004 as I am not one of the maintainers I have no idea. sorry.

**Comment by hohwille** [comment]
Sorry, M.I.A. - too many other OSS projects I am working on. Missed your PR. Thanks for it. Looks reasonable. I will review and merge within the next weeks...

**Comment by hohwille** [comment]
@andye2004 thanks for this useful improvement.
@matthiasbalke thanks for your review suggestion - I would have suggested the same to have some namespacing and avoid collision with other plugin properties.

**Comment by andye2004** [comment]
@hohwille Pleasure to help out! Could you give me an idea when it will be released? Would really like to swap out the local build we're using for the official one.

**Pull request merged**